source/reference/integration.rst

```diff
@@ -1,31 +1,159 @@
+.. module:: optuna.integration
+
 optuna.integration
 ==================
 
+The :mod:`~optuna.integration` module contains classes used to integrate Optuna with external machine learning frameworks.
```

Comment:
ditto

Reply:
This belongs to the reference part and I'll update later.